src/Activity.js

```diff
@@ -1,27 +1,22 @@
-const User = require('./User.js')
-const UserRepo = require('./userRepo.js')
-const userTestData = require('../test/userTestData.js')
-let allUsers = new UserRepo(userTestData);
 class Activity {
-    constructor(ActivityTestData){
+    constructor(ActivityTestData,user){
```

Comment:
Nice use of an extra parameter to get rid of those error message with files not needing require!